lib/manager/terraform/lockfile/index.ts

```diff
@@ -85,30 +86,23 @@ export async function updateArtifacts({
         ['provider', 'required_provider'].includes(dep.depType)
       );
       for (const dep of providerDeps) {
-        const lookupName = dep.lookupName ?? dep.depName;
+        massageProviderLookupName(dep);
```

Comment:
isn't this already done in extraction phase? Do we need to make this twice? 🤔

Reply:
Yes, should be. I have added this to be for sure backwards compatible.

Reply:
OK, so we should cleanup in a new PR. Looks pretty ugly now.